README.md

```diff
@@ -15,10 +15,12 @@ A Javascript SDK for receiving updates from the OpenSea Stream API - pushed over
 
 This is a best effort delivery messaging system. Messages that are not received due to connection errors will not be re-sent. Messages may be delievered out of order. This SDK is offered as a beta experience as we work with developers in the ecosystem to make this a more robust and reliable system.
 
-# Setup
+# Installation
 
-Run `nvm use`  
-And then `npm install`
+We recommend switching to Node.js version 16 to make sure common crypto dependencies work. Our minimum supported version is 16.11.0.
+
+- If you have Node Version Manager, run `nvm use` to ensure a compatible version is being used.
+- Install this package with `yarn install --save @opensea/stream-api`
```

Comment:
Think the command should be `yarn add --dev @opensea/stream-api`

Reply:
updated